service/host.go

```diff
@@ -278,9 +285,12 @@ func (s *host) start() Control {
 
 				s.shutdownMu.Unlock()
 				if _, err := s.shutdown(e, "", nil); err != nil {
-					s.Logger().Error("Unable to shut down modules", "initialError", e, "shutdownError", err)
+					s.Logger().Error("Unable to shut down modules",
+						zap.String("initialError", e.Error()),
+						zap.String("shutdownError", err.Error()),
```

Comment:
Possibly. I'll take another pass to make sure we don't lose any info. 

I'm not going to take a dependency on Daves error package in zap (it's pre-1.0), but I should still be able to support these errors with `Error`.